src/handler/s3Event.ts

```diff
@@ -1,23 +1,36 @@
 /* eslint-disable security/detect-non-literal-fs-filename */
 import * as fs from 'fs';
 import type { S3Event, S3EventRecord } from 'aws-lambda';
-import { configureFile } from '../fileConvert/fileConvert';
+import { configureTflFile } from '../fileConvert/fileConvert';
 import { filePull } from '../filePull/fromS3';
 import { filePush } from '../filePush/filePush';
 import { randomUUID } from 'crypto';
 import logger from '../util/logger';
 
-const handleEvent = async (record: S3EventRecord) => {
-  const workingDir = `/tmp/${randomUUID()}/`;
+const handleEvlEvent = async (record: S3EventRecord) => {
+  const workingDir = `/tmp/evl/${randomUUID()}/`;
   try {
-    fs.mkdirSync(workingDir);
+    fs.mkdirSync(workingDir, { recursive: true });
     const evlFileData = await filePull(record);
-    const filepath = await configureFile(
+    const filepath = await configureTflFile(
```

Comment:
Does this method want renaming to `configureTflFile`

Reply:
I really worry about myself sometimes

Reply:
😆